### DISCUSSION TITLE
Como ejecutar un archivo .c desde Git Bash

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#28] [Preguntas y respuestas] [answered]
Holaa

Logre crear un archivo .c con el ejemplo de "Hola Mundo" pero  no entiendo de donde o como compilarlo.
Gracias!

**Comment by martinvilu** [comment, marked as answer]
Con una terminal en el mismo lugar (donde haya sido hecho el `git clone` del repositorio)

Y si el compilador está instalado, ejecutar `gcc nombre.c` debiera de compilarlo, esto generará un archivo `a.exe` que podés ejecutar.

Puede funcionar `./a.exe` o `a.exe`, técnicamente si le haces doble click también funciona, pero va a funcionar tan rápido que vas a ver un destello negro de la consola por un instante nada más.

**Reply by agusscarava** [reply]
AA okok, no entendía lo del archivo a.exe
Tampoco me reconocía el comando `gcc`, pero ya pude no se como jaja, hice `gcc` y me genero un archivo .exe con el mismo nombre que le puse al archivo original y lo ejecuto con `./ejercicio1.exe` por ejemplo.
Gracias Martin!
